includes/ucf-news-config.php

```diff
@@ -55,6 +55,9 @@ public static function format_options( $list ) {
 					case 'limit':
 						$list[$key] = intval( $val );
 						break;
+					case 'per_row':
+						$per_row[$key] = intval( $val );
```

Comment:
`$per_row[$key]` needs to be `$list[$key]` here.  Alternatively, you could shorten this by including `per_row` as another possible value within the first switch statement, since they perform the same operation, e.g.:
```
case 'limit':
case 'per_row':
    $list[$key] = intval( $val );
    break;   
```